model/src/main/kotlin/Snippet.kt

```diff
@@ -17,10 +17,8 @@
  * License-Filename: LICENSE
  */
 
-package org.ossreviewtoolkit.model.utils
+package org.ossreviewtoolkit.model
```

Comment:
Commit message nit: The breaking api change should be indicated by an exclamation mark: "fix(model)!: ..."